discordrb.json

```diff
@@ -57,6 +57,28 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "userLoginWithPassword": "yes",
+            "userLoginWithToken": "yes",
+            "botLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "automatic",
+            "apiDocsUrl": "http://www.rubydoc.info/gems/discordrb",
+            "tutorials": "yes",
+            "tutorialsUrl": "https://github.com/meew0/discordrb/wiki",
+            "examples": "yes",
```

Comment:
Whether this lib has examples available, i. e. full runnable scripts that demonstrate a particular feature.